doc/dev/release.md

```diff
@@ -307,11 +308,28 @@ OPERATORSUM="$(sha256sum operator-sdk | cut -d ' ' -f 1)"
 brew bump-formula-pr --strict --url=$OPERATORSDKURL --sha256=$OPERATORSUM operator-sdk
 ```
 
-Note: If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
+**NOTE**  If there were any changes made to the CLI commands, make sure to look at the existing tests, in case they need updating.
 
 You've now fully released a new version of the Operator SDK. Good work! Make sure to follow the post-release steps below.
 
-### (Post-release) Updating the operator-sdk-samples repo
+### 6. Create JIRA tasks for the post-release actions
+
+After we do the release the following post-release actions are required to attend the users needs. So, create a JIRA for the post release actions  and ping them in the channel in order to allow other sign themselves to help with.
+
+**NOTE** You are free to working on in the post-release tasks if you wish as well. 
+
```

Comment:
However, the goal of this change was to add the step to create the JIRAs. So, if we remove it has no need to do this PR at all. We can just close it. wdyt?

Reply:
I would at least move this section to the post-release steps, and reword it so that it is a section description:

## Post-release steps

Each of the following steps should have an associated JIRA ticket assigned to either yourself or another team member.

### 1. Update the operator-sdk-samples repo 

...

Reply:
Done.